_plugins/category_generator.rb

```diff
@@ -19,8 +19,28 @@
 # - category_title_prefix: The string used before the category name in the page title (default is
 #                          'Category: ').
 
+PostCategory = Struct.new(:name, :dir) do
+  def to_liquid
+    { 'name' => name, 'dir' => dir }
```

Comment:
Here, let me save you hours:

* by default `to_liquid` is called on these objects
* if you return a hash with symbol keys, nothing will work and you'll be so mad

So just def this method and be sure to use string keys. 🥂